src/Tribe/Context/locations.php

```diff
@@ -18,6 +17,21 @@
 			}
 		],
 	],
+	'plain_permalink' => [
+		'read'  => [
+			Tribe__Context::FUNC => static function () {
```

Comment:
Since we're dealing with an option that will be fetched, by WordPress, on each request, does it make sense to cache?
I think the code could be simplified to this:
```php
'permalink_structure' => [
	'read' => [
		Tribe__Context::OPTION => [ 'permalink_structure' ],
	],
],
'plain_permalink' => [
	'read' => [
		Tribe__Context::LOCATION_FUNC => [
			'permalink_structure',
			static function( $struct ){
				return empty( $struct );
			},
		 ],
	],
],
```

This would add the `permalink_structure` and the `plain_permalink` locations and make the latter depend on the first.

Reply:
Oh I like that quite a bit.